recipes/scikit-explain/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "scikit-explain" %}
+{% set version = "0.0.9" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: e3601b24cbe3f862bfa4d8d8ee84c97d6990d53ba92e12da445f916ffa9c04d5
+
+build:
+  noarch: python 
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+
+  run:
+    - ipywidgets
+    - lime >=0.2.0
+    - matplotlib-base
+    - numpy
+    - pandas
+    - python >=3.8
+    - scikit-learn >=1.0.0
+    - seaborn >=0.11.0
+    - shap >=0.30.0
+    - statsmodels
+    - tqdm
+    - xarray >=0.16.0
+
+test:
+  imports:
+    - skexplain
+    - skexplain.common
+    - skexplain.main
+    - skexplain.main.PermutationImportance
+    - skexplain.plot
+
+about:
+  home: "https://github.com/monte-flora/scikit-explain/"
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: "A user-friendly python package for computing and plotting machine learning explainability output."
+  doc_url: "https://scikit-explain.readthedocs.io/en/latest/index.html#"
```

Comment:
this could just be `https://scikit-explain.readthedocs.io`

Reply:
Done.